api/WebSocket.json

```diff
@@ -159,10 +159,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "7"
```

Comment:
I don't totally trust that all of these attributes were introduced at the same time, but after the interface itself. Can you test an instance of `WebSocket` to make sure?

Reply:
Sure thing -- had the same issue with Chrome data for WebSocket, so I'm less trusting of the Safari data.  I'll close this PR and reopen it when we have a custom test!